met-api/src/met_api/services/email_verification_service.py

```diff
@@ -186,27 +189,26 @@ def _render_subscribe_email_template(survey: SurveyModel, token, subscription_ty
         return subject, body, args, template_id
 
     @staticmethod
-    def _render_survey_email_template(survey: SurveyModel, token):
+    def _render_survey_email_template(survey: SurveyModel, token, lang_code):
         # url is origin url excluding context path
         engagement: EngagementModel = EngagementModel.find_by_id(survey.engagement_id)
-        engagement_name = engagement.name
```

Comment:
I can see why you removed this. Not sure why we'd need a dedicated variable for it.